test/govuk_component/task_list_test.rb

```diff
@@ -5,80 +5,93 @@ def component_name
     "task_list"
   end
 
-  def simple_tasklist
+  def tasklist
     [
       [
         {
-          title: 'First header',
-          panel: 'First panel'
-        }
-      ],
-      [
-        {
-          title: 'Second header',
-          panel: 'Second panel'
-        }
-      ]
-    ]
-  end
-
-  def complex_tasklist
-    [
-      [
-        {
-          title: 'First group first step',
-          panel: '<p class="firstpanel">First panel</p>',
-          panel_descriptions: [
-            'First group first step first panel description',
-            'First group first step second panel description'
-          ],
-          panel_links: [
+          title: 'Group 1 step 1',
+          optional: true,
+          contents: [
             {
-              href: '/notalink1',
-              text: 'First group first step first panel link'
+              type: 'paragraph',
+              text: 'Group 1 step 1 paragraph'
             },
             {
-              href: '/notalink2',
```

Comment:
I saw a conversation about href vs path somewhere else, is there a reason to prefer one over the other?

Reply:
We should be consistent but I don't think there's any reason other than that. I started with path and then decided to switch to href, I just haven't done it yet.

Reply:
Updated, thanks for the reminder :)

Reply:
(in this case, these can presumably contain things other than paths, so href is more correct, unless you switch to something like url/uri)